frontend-pwa/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
```

Comment:
Can you add these instructions on how to run the webapp into a readme md file in frontend-pwa?